Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -127,6 +127,25 @@ div.bocList table.bocListTable td.bocListTitleCell > span
   padding: calc(var(--remotion-themed-spacing) / 2) var(--remotion-themed-spacing);
 }
 
+div.bocList table.bocListTable th.bocListTitleCell > a > img,
+div.bocList table.bocListTable th.bocListTitleCell > span > img
+{
+  width: 16px;
+  height: 16px;
+  margin-right: var(--remotion-themed-spacing); /* Provides a spacing between icon and text, which is reset below if only an icon is displayed */
+}
+
+div.bocList table.bocListTable th.bocListTitleCell > a > img:has(+ span.screenReaderText),
+div.bocList table.bocListTable th.bocListTitleCell > span > img:has(+ span.screenReaderText)
```

Comment:
```suggestion
div.bocList table.bocListTable th.bocListTitleCell > a > img:has(+ span.screenReaderText),
div.bocList table.bocListTable td.bocListTitleCell > a > img:has(+ span.screenReaderText),
div.bocList table.bocListTable th.bocListTitleCell > span > img:has(+ span.screenReaderText),
div.bocList table.bocListTable td.bocListTitleCell > span > img:has(+ span.screenReaderText)
```

Reply:
We don't need this style on the `td`s? This is the spacing between column title icon and column title text.

Reply:
please re-read: that's TD with class "bocListTitleCell" => we have both TH and TD elements in the header row of the BocList. It's for reasons of how screenreaders interpret column headers.